James Zou Profile Stanford Profiles Stanford Profiles Menu Browse Schools Graduate School of Business Graduate School of Education School of Earth Energy and Environmental Sciences School of Engineering School of Humanities and Sciences School of Medicine Institutes Research Centers Freeman Spogli Institute for International Studies Institute for Computational and Mathematical Engineering ICME Institute for Stem Cell Biology and Regenerative Medicine Precourt Institute for Energy SLAC National Accelerator Laboratory Stanford Stanford Institute for Economic Policy Research SIEPR Stanford PULSE Institute Stanford Woods Institute for the Environment Wu Tsai Neurosciences Institute Administration Office of VP for University Human Resources Office of Vice President for Business Affairs and Chief Financial Officer Public Affairs Stanford University Libraries Vice Provost and Dean of Research Vice Provost for Student Affairs Vice Provost for Undergraduate Education Sign In Edit My Profile James Zou Assistant Professor of Biomedical Data Science and by courtesy of Computer Science and of Electrical Engineering Web page https Print Profile Email Profile View Profile Tab Menu Bio Research Scholarship Teaching Publications Bio I am an Assistant Professor of Biomedical Data Science and by courtesy of Computer Science and Electrical Engineering at Stanford University I work on a wide range of problems in machine learning from proving mathematical properties to building algorithms and am especially interested in applications in biotech and health I received a from Harvard in 2014 and was at one time a member of Microsoft Research a Gates Scholar at the University of Cambridge and a Simons fellow at Berkeley I joined Stanford in 2016 and am excited to be an inaugural Investigator and the faculty director of the AI for Health program We are also a part of the Stanford AI Lab My research is supported by the NSF CAREER Award and the Google and Tencent AI awards Academic Appointments Assistant Professor Biomedical Data Science Assistant Professor By courtesy Computer Science Assistant Professor By courtesy Electrical Engineering Member Member Wu Tsai Neurosciences Institute Contact Academic jamesz University Faculty Department nbspBiomedical Data Science Position Asst Professor Additional Info Mail Code 5464 Links My website Twitter Current Research and Scholarly Interests My group works on both foundations of statistical machine learning and applications in biomedicine and healthcare We develop new technologies that make ML more accountable to humans more and reveals core scientific want our ML to be impactful and beneficial and as such we are deeply motivated by transformative applications in biotech and health We collaborate with and advise many academic and industry groups Courses Deep Learning in Genomics and BiomedicineBIODS 237 BIOMEDIN 273B CS 273B GENE 236 Aut Race Data Algorithms and HealthBIODS 240 Aut Topics in Biomedical Data Science inferenceBIODS 215 Win Value of Data and AICS 320 Win Workshop in BiostatisticsBIODS 260C STATS 260C Spr Independent Studies 13 Advanced Reading and ResearchCS 499 Aut Win Spr Sum Advanced Reading and ResearchCS 499P Aut Win Spr Sum Biomedical Informatics Teaching MethodsBIOMEDIN 290 Aut Win Spr Curricular Practical TrainingCS 390A Aut Win Spr Sum Curricular Practical TrainingCS 390B Win Directed Reading and ResearchBIODS 299 Aut Win Spr Sum Directed Reading and ResearchBIOMEDIN 299 Aut Win Spr Independent ProjectCS 399 Aut Independent WorkCS 199P Aut Win Spr Medical Scholars ResearchBIOMEDIN 370 Aut Win Spr Curricular Practical TrainingCS 390D Aut Win Senior ProjectCS 191 Aut Win Spr Writing Intensive Senior Project WIM CS 191W Aut Win Spr Prior Year Courses Courses AI Interpretability and FairnessCS 81SI Spr Data science and AI for 472 Spr Deep Learning in Genomics and BiomedicineBIODS 237 BIOMEDIN 273B CS 273B GENE 236 Aut Topics in Biomedical Data Science inferenceBIODS 215 Win Value of Data and AICS 320 Win Workshop in BiostatisticsBIODS 260C STATS 260C Spr Courses Deep Learning in Genomics and BiomedicineBIODS 237 BIOMEDIN 273B CS 273B GENE 236 Aut Workshop in BiostatisticsBIODS 260B STATS 260B Win Courses Deep Learning in Genomics and BiomedicineBIODS 237 BIOMEDIN 273B CS 273B GENE 236 Aut Topics in Biomedical Data Science inferenceBIODS 215 Win Workshop in BiostatisticsBIODS 260B STATS 260B Win Stanford Advisees Doctoral Dissertation Reader AC Louis Blankemeier Jiaqi Jiang Ismael Lemhadri Greg McInnes Stephen Pfohl Meltem Tolunay Jessica Torres Orals Chair Katherine McNamara Postdoctoral Faculty Sponsor Roxana Daneshjou Doctoral Dissertation Advisor AC Abubakar Abid Amirata Ghorbani Tony Ginart Ruishan Liu Jaime Roquero Gimenez Zhenqin Wu Orals Evaluator Anthony Corso Master Program Advisor Akash Chaurasia Soham Gadgil Albert Pun Anthony Tzen Alexander Verge Yuhui Zhang Doctoral Program Lingjiao Chen Bryan He John Hughes Garrett Thomas Kevin Wu Postdoctoral Research Mentor Yongchan Kwon All Publications Integrating spatial gene expression and breast tumour morphology via deep learning Nature biomedical engineering He Bergenstrahle Stenbeck Abid Andersson Borg Maaskola Lundeberg Zou J 2020 Abstract Spatial transcriptomics allows for the measurement of RNA abundance at a high spatial resolution making it possible to systematically link the morphology of cellular neighbourhoods and spatially localized gene expression Here we report the development of a deep learning algorithm for the prediction of local gene expression from histopathology images using a new dataset of spatially resolved gene expression data matched to histopathology images from 23 patients with breast cancer We identified over 100 genes including known breast cancer biomarkers of intratumoral heterogeneity and the of tumour growth and immune activation the expression of which can be predicted from the histopathology images at a resolution of 100m We also show that the algorithm generalizes well to The Cancer Genome Atlas and to other breast cancer gene expression datasets without the need for Predicting the spatially resolved transcriptome of a tissue directly from tissue images may enable screening for molecular biomarkers with spatial variation View details for DOI View details for PubMedID 32572199 AI for assessment of cardiac function Nature Ouyang He Ghorbani Yuan Ebinger Langlotz Heidenreich Harrington Liang Ashley Zou Y 2020 580 7802 Abstract Accurate assessment of cardiac function is crucial for the diagnosis of cardiovascular disease1 screening for cardiotoxicity2 and decisions regarding the clinical management of patients with a critical illness3 However human assessment of cardiac function focuses on a limited sampling of cardiac cycles and has considerable variability despite years of Here to overcome this challenge we present a deep learning surpasses the performance of human experts in the critical tasks of segmenting the left ventricle estimating ejection fraction and assessing cardiomyopathy Trained on echocardiogram videos our model accurately segments the left ventricle with a Dice similarity coefficient of predicts ejection fraction with a mean absolute error of and reliably classifies heart failure with reduced ejection fraction area under the curve of In an external dataset from another healthcare system predicts the ejection fraction with a mean absolute error of and classifies heart failure with reduced ejection fraction with an area under the curve of Prospective evaluation with repeated human measurements confirms that the model has variance that is comparable to or less than that of human experts By leveraging information across multiple cardiac cycles our model can rapidly identify subtle changes in ejection fraction is more reproducible than human evaluation and lays the foundation for precise diagnosis of cardiovascular disease in real time As a resource to promote further innovation we also make publicly available a large dataset of annotated echocardiogram videos View details for DOI View details for PubMedID 32269341 How Much Does Your Data Exploration Overfit Controlling Bias via Information Usage IEEE TRANSACTIONS ON INFORMATION THEORY Russo Zou J 2020 66 1 View details for DOI View details for Web of Science ID 000505566100019 Fast and method amplifies detection power in multiplehypothesis testing Nature communications Zhang Xia Zou J 2019 10 1 3433 Abstract Multiple hypothesis testing is an essential component of modern data science In many settings in addition to the additional covariates for each hypothesis are available functional annotation of variants in association studies Such information is ignored by popular multiple testing approaches such as the procedure BH Here we introduce AdaFDR a fast and flexible method that adaptively learns the optimal threshold from covariates to significantly improve detection power On eQTL analysis of the GTEx data AdaFDR discovers 32 more associations than BH at the same false discovery rate We prove that AdaFDR controls false discovery proportion and show that it makes substantially more discoveries while controlling false discovery rate FDR in extensive experiments AdaFDR is computationally efficient and allows covariates with both numeric and categorical values making it broadly useful across many applications View details for DOI View details for PubMedID 31366926 Large dataset enables prediction of repair after editing in primary T cells Nature biotechnology Leenay Aghazadeh Hiatt Tse Roth Apathy Shifrut Hultquist Krogan Wu Cirolia Canaj Leonetti Marson May Zou J 2019 Abstract Understanding of repair outcomes after DNA cleavage is still limited especially in primary human cells We sequence repair outcomes at genomic sites in primary human T cells and use these data to train a machine learning model which we have called CRISPR Repair Outcome SPROUT SPROUT accurately predicts the length probability and sequence of nucleotide insertions and deletions and will facilitate design of SpCas9 guide RNAs in therapeutically important primary human cells View details for DOI View details for PubMedID 31359007 Making AI Forget You Data Deletion in Machine Learning Ginart A Guan Valiant Zou Wallach Larochelle Beygelzimer Fox Garnett NEURAL INFORMATION PROCESSING SYSTEMS NIPS 2019 View details for Web of Science ID 000534424303050 Interpretation of Neural Networks Is Fragile Ghorbani Abid Zou AAAI ASSOC ADVANCEMENT ARTIFICIAL INTELLIGENCE 2019 View details for Web of Science ID 000485292603086 Clinical Genetics Lacks Standard Definitions and Protocols for the Collection and Use of Diversity Measures American journal of human genetics Popejoy A Crooks Fullerton Hindorff Hooker Koenig B Pino Ramos Ritter Wand Wright Yudell Zou Plon Bustamante Ormond Clinical Genome Resource ClinGen Ancestry and Diversity Working Group 2020 Abstract Genetics researchers and clinical professionals rely on diversity measures such as race ethnicity and ancestry REA to stratify study participants and patients for a variety of applications in research and precision medicine However there are no comprehensive widely accepted standards or guidelines for collecting and using such data in clinical genetics practice Two research consortia the Clinical Genome Resource ClinGen and Clinical Sequencing Research CSER have partnered to address this issue and report how REA are currently collected conceptualized and used Surveying clinical genetics professionals and researchers n 448 we found heterogeneity in the way REA are perceived defined and measured with variation in the perceived importance of REA in both clinical and research settings The majority of respondents 55 felt that REA are at least somewhat important for clinical variant interpretation ordering genetic tests and communicating results to patients However there was no consensus on the relevance of REA including how each of these measures should be used in different scenarios and what information they can convey in the context of human genetics A lack of common definitions and applications of REA across the precision medicine pipeline may contribute to inconsistencies in data collection missing or inaccurate classifications and misleading or inconclusive results Thus our findings support the need for standardization and harmonization of REA data collection and use in clinical genetics and precision health research View details for DOI View details for PubMedID 32504544 predicts RNA subcellular localization and highlights the role of splicing RNA New York Wu Parker Fazal Chang Zou J 2020 Abstract Subcellular localization is essential to RNA biogenesis processing and function across the gene expression life cycle However the specific nucleotide sequence motifs that direct RNA localization are incompletely understood Fortunately new sequencing technologies have provided atlases of RNA localization creating an opportunity to leverage computational modeling Here we present a new machine learning model that uses features to predict RNA localization across 8 different subcellular locations the first to provide such a wide range of predictions design enables high throughput sequence ablation and feature importance analyses to probe the sequence motifs that drive localization prediction We find localization informative motifs to be concentrated on 3 UTRs and scattered along the coding sequence and motifs related to splicing to be important drivers of predicted localization even for cytotopic distinctions for membraneless bodies within the nucleus or for organelles within the cytoplasm Overall our results suggest transcript splicing is one of many elements influencing RNA subcellular localization View details for DOI View details for PubMedID 32220894 AI for assessment of cardiac function NATURE Ouyang He Ghorbani Yuan Ebinger Langlotz Heidenreich Harrington Liang Ashley Zou Y 2020 View details for DOI View details for Web of Science ID 000521531000001 A benchmark of algorithms for the analysis of pooled CRISPR screens Genome biology Bodapati Daley Lin Zou Qi 2020 21 1 62 Abstract pooled knockout activation and repression screens are powerful tools for functional genomic investigations Despite their increasing importance there is currently little guidance on how to design and analyze screens Here we provide a review of the commonly used algorithms in the computational analysis of pooled CRISPR screens We develop a comprehensive simulation framework to benchmark and compare the performance of these algorithms using both synthetic and real datasets Our findings inform parameter choices of CRISPR screens and provide guidance to researchers on the design and analysis of pooled CRISPR screens View details for DOI View details for PubMedID 32151271 LitGen Genetic Literature Recommendation Guided by Human Explanations Pacific Symposium on Biocomputing Pacific Symposium on Biocomputing Nie Pineda Wright Wand Wulf Costa Patel Bustamante Zou J 2020 25 Abstract As genetic sequencing costs decrease the lack of clinical interpretation of variants has become the bottleneck in using genetics data A major rate limiting step in clinical interpretation is the manual curation of evidence in the genetic literature by highly trained biocurators What makes curation particularly is that the curator needs to identify papers that study variant pathogenicity using different types of approaches and biochemical assays or case control analysis In collaboration with the Clinical Genomic Resource ClinGen flagship NIH program for clinical propose the first machine learning system LitGen that can retrieve papers for a particular variant and filter them by specific evidence types used by curators to assess for pathogenicity LitGen uses deep learning to predict the type of provided by each paper It is trained on papers annotated by ClinGen curators and systematically evaluated on new test data collected by ClinGen LitGen further leverages rich human explanations and unlabeled data to gain relative performance improvement over models learned only on the annotated papers It is a useful framework to improve clinical variant curation View details for PubMedID 31797587 NCI Workshop on Artificial Intelligence in Radiation Oncology Training the Next Generation Practical radiation oncology Kang Thompson Aneja Lehman Trister Zou Obcemea El Naqa I 2020 Abstract Artificial intelligence AI is about to touch every aspect of radiotherapy from consultation treatment planning quality assurance therapy delivery to outcomes modeling There is an urgent need to train radiation oncologists and medical physicists in data science to help shepherd AI solutions into clinical practice Poorly trained personnel may do more harm than good when attempting to apply rapidly developing and complex technologies As the amount of AI research expands in our field the radiation oncology community needs to discuss how to educate future generations in this area The National Cancer Institute NCI Workshop on AI in Radiation Oncology Shady Grove MD April 2019 was the first https of two data science workshops in radiation oncology hosted by the NCI in 2019 During this workshop the Training and Education Working Group was formed by volunteers among the invited attendees Its members represent radiation oncology medical physics radiology computer science industry and the NCI In this perspective article written by members of the Training and Education Working Group we provide and discuss Action Points relevant for future trainees interested in radiation oncology AI 1 creating AI awareness and responsible conduct 2 implementing a practical didactic curriculum 3 creating a publicly available database of training resources and 4 accelerate learning and funding opportunities Together these Action Points can facilitate the translation of AI into clinical practice View details for DOI View details for PubMedID 32544635 Predicting target genes of noncoding regulatory variants with ICE Bioinformatics Oxford England Wu Ioannidis Zou J 2020 Abstract Interpreting genetic variants of unknown significance VUS is essential in clinical applications of genome sequencing for diagnosis and personalized care Noncoding variants remain particularly difficult to interpret despite making up a large majority of trait associations identified in GWAS analyses Predicting the regulatory effects of noncoding variants on candidate genes is a key step in evaluating their clinical significance Here we develop a machine learning algorithm ICE Inference of Connected eQTLs to predict the regulatory targets of noncoding variants identified in studies of expression quantitative trait loci eQTLs We assemble datasets using eQTL results from the Expression GTEx project and learn to separate positive and negative pairs based on annotations characterizing the variant gene and the intermediate sequence ICE achieves an area under the receiver operating characteristic curve of using random and for a more stringent Further evaluation on rare variants and regulatory variants shows a significant enrichment in ICE identifying the true target genes versus negative controls In gene ranking experiments ICE achieves a accuracy of 50 and accuracy of 90 Salient features including GC content histone modifications and interactions are further analyzed and visualized to illustrate their influences on predictions ICE can be applied to any VUS of interest and each candidate nearby gene to output a score reflecting the likelihood of regulatory effect on the expression level These scores can be used to prioritize variants and genes to assist in patient diagnosis and GWAS and data used in this work are available at https data View details for DOI View details for PubMedID 32330225 Automatic peak integration by sequential deep learning for multiple reaction monitoring Journal of proteomics Wu Serie Xu Zou J 2020 103820 Abstract Mass spectrometry MS based proteomics has become an indispensable component of modern molecular and cellular biochemistry analysis Multiple reaction monitoring MRM is one of the most MS techniques for molecule detection and quantification Despite its wide usage there lacks an accurate computational framework to analyze MRM data and expert annotation is often required especially to perform peak integration Here we propose a deep learning method Peak Boundary Neural Network built upon recent advances in sequential neural networks for fully automatic chromatographic peak integration To train we generated a large dataset of over expert annotated peaks from MS transitions spanning a wide dynamic range including both peptides and intact glycopeptides Our model demonstrated outstanding performances on unseen test samples reaching agreement Pearson r with human annotated ground truth Systematic evaluations also show that is substantially more robust and accurate compared to previous peak integration software can benefit the wide community of mass spectrometry data analysis especially in applications involving MS experiments Codes and test data used in this work are available at https SIGNIFICANCE Human annotations serve an important role in accurate quantification of multiple reaction monitoring MRM experiments though they are costly to collect and limit analysis throughput In this work we proposed and developed a novel technique for the step in MRM based on recent innovations in sequential deep learning models We collected in total MRM peaks and trained a set of accurate and robust neural networks for the task Results demonstrated a substantial improvement over the current software for mass spectrometry analysis and comparable level of accuracy and precision as human annotators View details for DOI View details for PubMedID 32416316 Predicts RNA Localization to Host Mitochondria and Nucleolus bioRxiv the preprint server for biology Wu Zou Chang Y 2020 Abstract The coronavirus is driving a global pandemic but its biological mechanisms are less well understood is an RNA virus whose multiple genomic and subgenomic RNA sgRNA transcripts hijack the host cell machinery located across distinct cytotopic locations Subcellular localization of its viral RNA could play important roles in viral replication and host antiviral immune response Here we perform computational modeling of viral RNA localization across eight subcellular neighborhoods We compare hundreds of genomes to the human transcriptome and other coronaviruses and perform systematic analyses to identify the responsible signals Using machine learning models we predict that the RNA genome and all sgRNAs are enriched in the host mitochondrial matrix and nucleolus The 5 and 3 viral untranslated regions possess the strongest and most distinct localization signals We discuss the mitochondrial localization signal in relation to the formation of vesicles a critical stage in the coronavirus life cycle Our computational analysis serves as a hypothesis generation tool to suggest models for biology and inform experimental efforts to combat the virus View details for DOI View details for PubMedID 32511373 View details for PubMedCentralID PMC7263502 Deep learning interpretation of echocardiograms NPJ digital medicine Ghorbani Ouyang Abid He Chen Harrington Liang Ashley Zou Y 2020 3 10 Abstract Echocardiography uses ultrasound technology to capture high temporal and spatial resolution images of the heart and surrounding structures and is the most common imaging modality in cardiovascular medicine Using convolutional neural networks on a large new dataset we show that deep learning applied to echocardiography can identify local cardiac structures estimate cardiac function and predict systemic phenotypes that modify cardiovascular risk but not readily identifiable to human interpretation Our deep learning model EchoNet accurately identified the presence of pacemaker leads enlarged left atrium left ventricular hypertrophy left ventricular end systolic and diastolic volumes R 2 and R 2 and ejection fraction R 2 as well as predicted systemic phenotypes of age R 2 sex weight R 2 and height R 2 Interpretation analysis validates that EchoNet shows appropriate attention to key cardiac structures when performing tasks and highlights regions of interest when predicting systemic phenotypes difficult for human interpretation Machine learning on echocardiography images can streamline repetitive tasks in the clinical workflow provide preliminary interpretation in areas with insufficient qualified cardiologists and predict phenotypes challenging for human evaluation View details for DOI View details for PubMedID 31993508 Sex and gender analysis improves science and engineering Nature Tannenbaum Ellis Eyssel Zou Schiebinger 2019 575 7781 Abstract The goal of sex and gender analysis is to promote rigorous reproducible and responsible science Incorporating sex and gender analysis into experimental design has enabled advancements across many disciplines such as improved treatment of heart disease and insights into the societal impact of algorithmic bias Here we discuss the potential for sex and gender analysis to foster scientific discovery improve experimental efficiency and enable social equality We provide a roadmap for sex and gender analysis across scientific disciplines and call on researchers funding agencies journals and universities to coordinate efforts to implement robust methods of sex and gender analysis View details for DOI View details for PubMedID 31695204 VetTag improving automated veterinary diagnosis coding via language modeling NPJ DIGITAL MEDICINE Zhang Nie Zehnder Page Zou J 2019 2 View details for DOI View details for Web of Science ID 000467504500001 Modeling Spatial Correlation of Transcripts with Application to Developing Pancreas SCIENTIFIC REPORTS Liu Mignardi Jones Enge Kim Quake Zou J 2019 9 View details for DOI View details for Web of Science ID 000463178500053 Modeling Spatial Correlation of Transcripts with Application to Developing Pancreas Scientific reports Liu Mignardi Jones Enge Kim Quake Zou J 2019 9 1 5592 Abstract Recently transcriptomic methods were developed and enabled researchers to spatially resolve gene expression variation at the molecular level for the first time In this work we develop a general analysis tool to quantitatively study the spatial correlations of gene expression in fixed tissue sections As an illustration we analyze the spatial distribution of single mRNA molecules measured by in situ sequencing on human fetal pancreas at three developmental time 87 and 117days We develop a density method to capture the spatial relationship between gene expression and other morphological features of the tissue sample such as position of nuclei and endocrine cells of the pancreas In addition we build a statistical model to characterize correlations in the spatial distribution of the expression level among different genes This model enables us to infer the inhibitory and clustering effects throughout different time points Our analysis framework is applicable to a wide variety of transcriptomic data to derive biological insights View details for PubMedID 30944357 VetTag improving automated veterinary diagnosis coding via language modeling NPJ digital medicine Zhang Nie Zehnder Page Zou J 2019 2 35 Abstract Unlike human medical records most of the veterinary records are free text without standard diagnosis coding The lack of systematic coding is a major barrier to the growing interest in leveraging veterinary records for public health and translational research Recent machine learning effort is limited to predicting 42 diagnosis categories from veterinary notes Here we develop a algorithm to automatically predict all 4577 standard veterinary diagnosis codes from free text We train our algorithm on a curated dataset of over 100 K expert labeled veterinary notes and over one million unlabeled notes Our algorithm is based on the adapted Transformer architecture and we demonstrate that language modeling on the unlabeled notes via pretraining and as an auxiliary objective during supervised learning greatly improves performance We systematically evaluate the performance of the model and several baselines in challenging settings where algorithms trained on one hospital are evaluated in a different hospital with substantial domain shift In addition we show that hierarchical training can address severe data imbalances for diagnosis with a few training cases and we provide interpretation for what is learned by the deep network Our algorithm addresses an important challenge in veterinary medicine and our model and experiments add insights into the power of unsupervised learning for clinical natural language processing View details for DOI View details for PubMedID 31304381 View details for PubMedCentralID PMC6550141 Contingent Payment Mechanisms for Resource Utilization Ma Meir Parkes Zou Assoc Comp Machinery ASSOC COMPUTING MACHINERY 2019 View details for Web of Science ID 000474345000052 Knockoffs for the Mass New Feature Importance Statistics with False Discovery Guarantees Gimenez Ghorbani Zou Chaudhuri Sugiyama MICROTOME PUBLISHING 2019 View details for Web of Science ID 000509687902018 A large bystander mutation causes immune dysregulation Communications biology Simeonov Brandt Chan Cortez Li Woo Lee Carvalho Indart Roth Zou May Lupski Anderson Buaas Rokhsar Marson A 2019 2 70 Abstract A persistent concern with gene editing has been the potential to generate mutations at genomic sites While mice to delete a intronic enhancer here we discovered a founder line that had marked immune dysregulation caused by a 24kb tandem duplication of the sequence adjacent to the deletion Our results suggest unintended repair of genomic cuts can cause pathogenic bystander mutations that escape detection by routine targeted genotyping assays View details for PubMedID 30793048 Contrastive Multivariate Singular Spectrum Analysis Dirie Abid Zou IEEE IEEE 2019 View details for Web of Science ID 000535355700155 Improving the Stability of the Knockoff Procedure Multiple Simultaneous Knockoffs and Entropy Maximization Gimenez Zou Chaudhuri Sugiyama MICROTOME PUBLISHING 2019 View details for Web of Science ID 000509687902024 A primer on deep learning in genomics Nature genetics Zou Huss Abid Mohammadi Torkamani Telenti A 2018 Abstract Deep learning methods are a class of machine learning techniques capable of identifying highly complex patterns in large datasets Here we provide a perspective and primer on deep learning applications for genome analysis We discuss successful applications in the fields of regulatory genomics variant calling and pathogenicity scores We include general guidance for how to effectively use deep learning methods as well as a practical guide to tools and resources This primer is accompanied by an interactive online tutorial View details for PubMedID 30478442 The clinical imperative for inclusivity Race ethnicity and ancestry REA in genomics Human mutation Popejoy A Ritter Crooks Currey Fullerton Hindorff Koenig Ramos Sorokin Wand Wright Zou Gignoux Bonham Plon Bustamante Clinical Genome Resource ClinGen Ancestry and Diversity Working Group ADWG 2018 39 11 Abstract The Clinical Genome Resource ClinGen Ancestry and Diversity Working Group highlights the need to develop guidance on race ethnicity and ancestry REA data collection and use in clinical genomics We present quantitative and qualitative evidence to characterize 1 acquisition of REA data via clinical laboratory requisition forms and 2 information disparity across populations in the Genome Aggregation Database gnomAD at clinically relevant sites ascertained from annotations in ClinVar Our requisition form analysis showed substantial heterogeneity in clinical laboratory ascertainment of REA as well as marked incongruity among terms used to define REA categories There was also striking disparity across REA populations in the amount of information available about clinically relevant variants in gnomAD European ancestral populations constituted the majority of observations allele counts and private alleles in gnomAD at 550 loci with pathogenic and likely pathogenic variants in ClinVar Our findings highlight the importance of implementing and supporting programs to increase diversity in genome sequencing and clinical genomics as well as measuring uncertainty around datasets that are used in variant interpretation Finally we suggest the need for a standardized REA data collection framework to be developed through partnerships and collaborations and adopted across clinical genomics View details for PubMedID 30311373 DeepTag inferring diagnoses from veterinary clinical notes NPJ DIGITAL MEDICINE Nie Zehnder Page Zhang Pineda Rivas Bustamante Zou J 2018 1 View details for DOI View details for Web of Science ID 000449685400001 Integrative proteomics and bioinformatic prediction enable a apicoplast proteome in malaria parasites PLoS biology Boucher Ghosh Zhang Lal Jang Ju Zhang Wang Ralph Zou Elias Yeh 2018 16 9 e2005895 Abstract Malaria parasites Plasmodium spp and related apicomplexan pathogens contain a nonphotosynthetic plastid called the apicoplast Derived from an unusual secondary endosymbiosis the apicoplast is a fascinating organelle whose function and biogenesis rely on a complex amalgamation of bacterial and algal pathways Because these pathways are distinct from the human host the apicoplast is an excellent source of novel antimalarial targets Despite its biomedical importance and evolutionary significance the absence of a reliable apicoplast proteome has limited most studies to the handful of pathways identified by homology to bacteria or primary chloroplasts precluding our ability to study the most novel apicoplast pathways Here we combine proximity proteomics BioID and a new machine learning algorithm to generate a apicoplast proteome consisting of 346 proteins Critically the high accuracy of this proteome significantly outperforms previous methods and extends beyond other BioID studies of unique parasite compartments Half of identified proteins have unknown function and 77 are predicted to be important for normal growth We validate the apicoplast localization of a subset of novel proteins and show that an cassette protein ABCF1 is essential for survival and plays a previously unknown role in apicoplast biogenesis These findings indicate critical organellar functions for newly discovered apicoplast proteins The apicoplast proteome will be an important resource for elucidating unique pathways derived from secondary endosymbiosis and prioritizing antimalarial drug targets View details for PubMedID 30212465 Design AI so that it fair NATURE Zou Schiebinger 2018 559 7714 View details for DOI View details for Web of Science ID 000439059800025 View details for PubMedID 30018439 Exploring patterns enriched in a dataset with contrastive principal component analysis NATURE COMMUNICATIONS Abid Zhang Bagaria Zou J 2018 9 2134 Abstract Visualization and exploration of data is a ubiquitous challenge across disciplines Widely used techniques such as principal component analysis PCA aim to identify dominant trends in one dataset However in many settings we have datasets collected under different conditions a treatment and a control experiment and we are interested in visualizing and exploring patterns that are specific to one dataset This paper proposes a method contrastive principal component analysis cPCA which identifies structures that are enriched in a dataset relative to comparison data In a wide variety of experiments we demonstrate that cPCA with a background dataset enables us to visualize patterns missed by PCA and other standard methods We further provide a geometric interpretation of cPCA and strong mathematical guarantees An implementation of cPCA is publicly available and can be used for exploratory data analysis in many applications where PCA is currently used View details for PubMedID 29849030 Word embeddings quantify 100 years of gender and ethnic stereotypes PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA Garg Schiebinger Jurafsky Zou J 2018 115 16 Abstract Word embeddings are a powerful framework that represents each English word by a vector The geometric relationship between these vectors captures meaningful semantic relationships between the corresponding words In this paper we develop a framework to demonstrate how the temporal dynamics of the embedding helps to quantify changes in stereotypes and attitudes toward women and ethnic minorities in the 20th and 21st centuries in the United States We integrate word embeddings trained on 100 y of text data with the US Census to show that changes in the embedding track closely with demographic and occupation shifts over time The embedding captures societal the women movement in the 1960s and Asian immigration into the United also illuminates how specific adjectives and occupations became more closely associated with certain populations over time Our framework for temporal analysis of word embedding opens up a fruitful intersection between machine learning and quantitative social science View details for PubMedID 29615513 DeepTag inferring diagnoses from veterinary clinical notes NPJ digital medicine Nie Zehnder Page Zhang Pineda Rivas Bustamante Zou J 2018 1 60 Abstract Large scale veterinary clinical records can become a powerful resource for patient care and research However clinicians lack the time and resource to annotate patient records with standard medical diagnostic codes and most veterinary visits are captured in notes The lack of standard coding makes it challenging to use the clinical data to improve patient care It is also a major impediment to translational research which relies on the ability to accurately identify patient cohorts with specific diagnostic criteria in humans and animals In order to reduce the coding burden for veterinary clinical practice and aid translational research we have developed a deep learning algorithm DeepTag which automatically infers diagnostic codes from veterinary notes DeepTag is trained on a newly curated dataset of veterinary notes manually annotated by experts DeepTag extends multitask LSTM with an improved hierarchical objective that captures the semantic structures between diseases To foster collaboration DeepTag also learns to abstain in examples when it is uncertain and defers them to human experts resulting in improved performance DeepTag accurately infers disease codes from even in challenging settings where the text comes from different clinical settings than the ones used for training It enables automated disease annotation across a broad range of clinical diagnoses with minimal preprocessing The technical framework in this work can be applied in other medical domains that currently lack medical coding resources View details for DOI View details for PubMedID 31304339 View details for PubMedCentralID PMC6550285 Embedding for Informative Missingness Deep Learning With Incomplete Data Ghorbani Zou IEEE IEEE 2018 View details for Web of Science ID 000461021200062 The Effects of Memory Replay in Reinforcement Learning Liu Zou IEEE IEEE 2018 View details for Web of Science ID 000461021200067 Autowarp Learning a Warping Distance from Unlabeled Time Series Using Sequence Autoencoders Abid Zou Bengio Wallach Larochelle Grauman CesaBianchi Garnett NEURAL INFORMATION PROCESSING SYSTEMS NIPS 2018 View details for Web of Science ID 000461852005015 Diabetes reversal by inhibition of the tyrosine phosphatase NATURE CHEMICAL BIOLOGY Stanford Aleshin Zhang Ardecky Hedrick Zou Ganji Bliss Yamamoto Bobkov A Kiselar Liu Cadwell Khare Yu Barquilla Chung Mustelin Schenk Bankston Liddington Pinkerton A Bottini 2017 13 6 Abstract insulin resistance plays a central role in type 2 diabetes As such tyrosine phosphatases that dephosphorylate the insulin receptor IR are potential therapeutic targets The protein tyrosine phosphatase LMPTP is a proposed IR phosphatase yet its role in insulin signaling in vivo has not been defined Here we show that global and LMPTP deletion protects mice from diabetes without affecting body weight To examine the role of the catalytic activity of LMPTP we developed a inhibitor with a novel uncompetitive mechanism a unique binding site at the opening of the catalytic pocket and an exquisite selectivity over other phosphatases This inhibitor is orally bioavailable and it increases liver IR phosphorylation in vivo and reverses diabetes Our findings suggest that LMPTP is a key promoter of insulin resistance and that LMPTP inhibitors would be beneficial for treating type 2 diabetes View details for DOI View details for Web of Science ID 000401419300015 View details for PubMedID 28346406 Correcting for heterogeneity in DNA methylation a comprehensive evaluation Nature methods Rahmani Zaitlen Baran Eng Hu Galanter Oh Burchard Eskin Zou Halperin 2017 14 3 View details for DOI View details for PubMedID 28245214 NeuralFDR Learning Discovery Thresholds from Hypothesis Features Xia Zhang Zou Tse Guyon Luxburg Bengio Wallach Fergus Vishwanathan Garnett NEURAL INFORMATION PROCESSING SYSTEMS NIPS 2017 View details for Web of Science ID 000452649401056 Quantifying unobserved variants in human populations provides a roadmap for sequencing projects Nature communications Zou Valiant Valiant Karczewski Chan Samocha Lek Sunyaev Daly MacArthur 2016 7 Abstract As new proposals aim to sequence ever larger collection of humans it is critical to have a quantitative framework to evaluate the statistical power of these projects We developed a new algorithm UnseenEst and applied it to the exomes of individuals to estimate the frequency distribution of all variants including rare variants that have not been observed yet in the current cohorts Our results quantified the number of new variants that we expect to identify as sequencing cohorts reach hundreds of thousands of individuals With 500K individuals we find that we expect to capture of all possible variants and 12 of all possible missense variants We also estimate that genes have frequency of in healthy humans consistent with very strong intolerance to gene inactivation View details for DOI View details for PubMedID 27796292 View details for PubMedCentralID PMC5095512 Limits on Active to Sterile Neutrino Oscillations from Disappearance Searches in the MINOS Daya Bay and Experiments PHYSICAL REVIEW LETTERS Adamson An Anghel Aurisano Balantekin A Band Barr Bishai Blake Blyth Bock Bogert Cao Cao Cao Cao Carroll Castromonte Cen CHAN Chang Chang Chang Chen Chen Chen Chen Chen Chen Cheng Cheng Cheng Cheng CHERWINKA Childress Chu Chukanov Coelho J Corwin Cummings de Arcos De Rijck Deng Devan Devenish Ding Ding Diwan Dolgareva Dove Dwyer Edwards Escobar Evans Falk FELDMAN Flanagan Frohne Gabrielyan Gallagher Germani Gill Gomes Gonchar Gong Gong Goodman Gouffon Graf Gran Grassi Grzelak Gu Guan Guo Guo Guo Guo Habig Hackenburg Hahn Han Hans Hartnell Hatcher He Heeger Heng Higuera Holin Hor Hsiung Y Hu B Hu Hu Huang Huang Huang Huang Huber Huo Hussain Hylen Irwin Isvan Jaffe Jaffke James Jen Jensen Jetter Ji Ji Jiao J Johnson de Jong Joshi Kafka Kang Kasahara Kettell Kohn Koizumi Kordosky Kramer Kreymer Kwan Kwok Kwok Lang Langford Lau Lebanowski Lee Lee Lei Leitner Leung Li Li Li Li Li Li LI Li Li Li Li Z Liang Lin Lin Lin Lin Lin Ling Link Litchfield Littenberg Littlejohn Liu Liu Liu Loh Lu Lu Lu Lucas Luk Lv Ma Ma X Ma Ma Malyshkin Mann Marshak Caicedo Mayer McDonald McGivern McKeown Medeiros Mehdiyev Meier Messier Miller Mishra Mitchell Mooney Moore Mualem Musser Nakajima Naples Napolitano Naumov Naumova Nelson Newman Ngai Nichol Ning Nowak J Olshevskiy Orchanian Pahlka Paley Pan Park Patterson Patton Pawloski Pec Peng Perch Pfuetzner Phan Pinsky Plunkett Poonthottathil Pun Qi Qi Qian Qiu Radovic Raper Rebel Ren Rosenfeld Rosero Roskovec Ruan Rubin Sail Sanchez Schneps Schreckenberger Schreiner Sharma Sher Sousa Steiner Sun Sun Tagg Talaga Tang Taychenachev Thomas Thomson Tian Timmons Todd Tognini Toner Torretta Treskov Tsang Tull Tzanakos Urheim Vahle Viaux Viren Vorobel Wang Wang Wang Wang Wang Wang Wang Wang Wang Webb Weber Wei WEN Whisnant White Whitehead Whitehead Wise Wojcicki Wong Wong Worcester Wu Wu Wu Xia Xia Xing Z Xu Xu Xu Xue Yang Yang Yang Yang Yang Ye Ye Yeh Young Yu Zeng Zhan Zhang Zhang Zhang Zhang Zhang Zhang Zhang Zhang Zhang Zhang Zhao Zhao Zhao Y Zhong Zhou Zhou Zhuang Zou 2016 117 15 Abstract Searches for a light sterile neutrino have been performed independently by the MINOS and the Daya Bay experiments using the muon anti neutrino and electron antineutrino disappearance channels respectively In this Letter results from both experiments are combined with those from the reactor neutrino experiment to constrain oscillations into light sterile neutrinos The three experiments are sensitive to complementary regions of parameter space enabling the combined analysis to probe regions allowed by the Liquid Scintillator Neutrino Detector LSND and MiniBooNE experiments in a minimally extended flavor framework Stringent limits on 2 μe are set over 6 orders of magnitude in the sterile splitting 41 2 The mixing phase space allowed by the LSND and MiniBooNE experiments is excluded for 41 2 2 at 95 s View details for DOI View details for PubMedID 27768356 Hierarchical Patterning of Multifunctional Conducting Polymer Nanoparticles as a Bionic Platform for Topographic Contact Guidance ACS NANO Ho Zou Chen Munshi Smith Agarwal Hodgetts Plant Bakker Harvey Luzinov Iyer 2015 9 2 Abstract The use of programmed electrical signals to influence biological events has been a widely accepted clinical methodology for neurostimulation An optimal biocompatible platform for neural activation efficiently transfers electrical signals across the interface and also incorporates neural guidance conduits Inherently conducting polymers ICPs have emerged as frontrunners as soft biocompatible alternatives to traditionally used metal electrodes which are highly invasive and elicit tissue damage over implantation However fabrication techniques for the ICPs suffer a major bottleneck which limits their usability and medical translation Herein we report that these limitations can be overcome using colloidal chemistry to fabricate multimodal conducting polymer nanoparticles Furthermore we demonstrate that these polymer nanoparticles can be precisely assembled into linear conduits using surface chemistry Finally we validate that this platform can act as guidance conduits for neurostimulation whereby the presence of electrical current induces remarkable dendritic axonal sprouting of cells View details for DOI View details for Web of Science ID 000349940500072 View details for PubMedID 25623615 Endovascular Repair With the Chimney Technique for Stanford Type B Aortic Dissection Involving Arch With Mirror Image Branching JOURNAL OF ENDOVASCULAR THERAPY Ma Yang Xu Zou Jiang Jiao Zhang X 2013 20 3 Abstract To report endovascular repair with the chimney technique of type B aortic dissection involving a aortic arch RAA hypertensive men aged 48 and 42 years with symptoms of aortic dissection resistant to medical therapy underwent emergent thoracic endovascular aortic repair with the chimney technique to extend the proximal landing zones Both patients had arches with mirror image branching One patient required a bare metal chimney stent to maintain perfusion to the right subclavian artery while the other patient had a chimney stent to revascularize the right common carotid artery 1 year and 1 month respectively showed that there was positive aortic remodeling and the chimney stents were TEVAR seems safe and effective for Stanford type B dissection in patients having RAA with mirror image branching and no sufficient proximal fixation zone View details for Web of Science ID 000320074100005 View details for PubMedID 23731297 Conversion of Human Fibroblasts to Functional Endothelial Cells by Defined Factors ARTERIOSCLEROSIS THROMBOSIS AND VASCULAR BIOLOGY Li Huang Zou Laurent Lee Okogbaa Cooke Ding 2013 33 6 Abstract Transdifferentiation of fibroblasts to endothelial cells ECs may provide a novel therapeutic avenue for diseases including ischemia and fibrosis Here we demonstrate that human fibroblasts can be transdifferentiated into functional ECs by using only 2 factors Oct4 and Klf4 under inductive signaling determine whether human fibroblasts could be converted into ECs by transient expression of pluripotency factors human neonatal fibroblasts were transduced with lentiviruses encoding Oct4 and Klf4 in the presence of soluble factors that promote the induction of an endothelial program After 28 days clusters of induced endothelial iEnd cells seemed and were isolated for further propagation and subsequent characterization The iEnd cells resembled primary human ECs in their transcriptional signature by expressing endothelial phenotypic markers such as CD31 vascular and von Willebrand Factor Furthermore the iEnd cells could incorporate acetylated lipoprotein and form vascular structures in vitro and in vivo When injected into the ischemic limb of mice the iEnd cells engrafted increased capillary density and enhanced tissue perfusion During the transdifferentiation process the endogenous pluripotency network was not activated suggesting that this process bypassed a pluripotent intermediate transdifferentiation can be successfully applied for generating functional autologous ECs for therapeutic applications View details for DOI View details for Web of Science ID 000319119500038 View details for PubMedID 23520160 Amino Acid Homeostasis Modulates Salicylic Redox Status and Defense Responses in Arabidopsis PLANT CELL Liu Ji Bhuiyan Pilot Selvaraj Zou Wei Y 2010 22 11 Abstract The tight association between nitrogen status and pathogenesis has been broadly documented in interactions However the interface between primary metabolism and disease responses remains largely unclear Here we show that knockout of a single amino acid transporter LYSINE HISTIDINE TRANSPORTER1 LHT1 is sufficient for Arabidopsis thaliana plants to confer a broad spectrum of disease resistance in a salicylic manner We found that redox in photosynthetic cells was causally linked to the lht1 phenotypes Furthermore the enhanced resistance in lht1 could be attributed to a specific deficiency of its main physiological substrate Gln and not to a general nitrogen deficiency Thus by enabling nitrogen metabolism to moderate the cellular redox status a plant primary metabolite Gln plays a crucial role in plant disease resistance View details for DOI View details for Web of Science ID 000285576500025 View details for PubMedID 21097712 View details for PubMedCentralID PMC3015111 Alcoholic neurobiology Changes in dependence and recovery 12th International Congress of the Crews Buckley Dodd Ende Foley Harper He Innes Loh Pfefferbaum Zou SULLIVAN 2005 Abstract This article presents the proceedings of a symposium held at the meeting of the International Society for Biomedical Research on Alcoholism ISBRA in Mannheim Germany in October 2004 Chronic alcoholism follows a fluctuating course which provides a naturalistic experiment in vulnerability resilience and recovery of human neural systems in response to presence absence and history of the neurotoxic effects of alcoholism Alcohol dependence is a progressive chronic disease that is associated with changes in neuroanatomy neurophysiology neural gene expression psychology and behavior Specifically alcohol dependence is characterized by a neuropsychological profile of mild to moderate impairment in executive functions visuospatial abilities and postural stability together with relative sparing of declarative memory language skills and primary motor and perceptual abilities Recovery from alcoholism is associated with a partial reversal of CNS deficits that occur in alcoholism The reversal of deficits during recovery from alcoholism indicates that brain structure is capable of repair and restructuring in response to insult in adulthood Indirect support of this repair model derives from studies of selective neuropsychological processes structural and functional neuroimaging studies and preclinical studies on degeneration and regeneration during the development of alcohol dependence and recovery form dependence Genetics and brain regional specificity contribute to unique changes in neuropsychology and neuroanatomy in alcoholism and recovery This symposium includes presentations on changes that occur during active alcoholism as well as those that may occur during from alcohol dependence Included are human neuroimaging and neuropsychological assessments changes in human brain gene expression allelic combinations of genes associated with alcohol dependence and preclinical studies investigating mechanisms of alcohol induced neurotoxicity and neuroprogenetor cell expansion during recovery from alcohol dependence View details for DOI View details for Web of Science ID 000231767900018 View details for PubMedID 16156047 PROTECT AGAINST LETHAL INFECTION AND LETHAL CHALLENGE IN MICE JOURNAL OF IMMUNOLOGY STARNES PEARCE Tewari Yim Zou Abrams 1990 145 12 Abstract Potentially fatal physiologic and metabolic derangements can occur in response to bacterial infection in animals and man Recently it has been shown that alterations in the levels of circulating cytokines such as and occur shortly after bacterial challenge To understand better the role of in inflammation we investigated the effects of in vivo antibody treatment in a mouse model of septic shock Rat neutralizing mAb was produced from splenocytes of an animal immunized with mouse This mAb was a very potent and specific antagonist of mouse in vitro bioactivity demonstrated using the NFS60 myelomonocytic and KD83 plasmacytoma target cell lines and also immunoprecipitated radiolabeled mAb pretreatment of mice subsequently challenged with lethal doses of Escherichia coli or protected mice from death caused by these treatments Pretreatment of mice with led to an increase in serum TNF bioactivity in comparison to isotype control antibody implicating as a negative modulator of TNF in vivo treatment of mice challenged with live coli resulted in a 70 decrease in serum levels determined by immunoenzymetric assay compared to control antibody thereby supporting a role for as a positive regulator of levels We conclude that is a mediator in lethal coli infection and suggest that antagonists of may be beneficial therapeutically in bacterial infection View details for Web of Science ID A1990EP04100033 View details for PubMedID 2124237 Report Accessibility Issues Powered By CAP Network Stanford University SU Home Maps Directions Search Stanford Terms of Use Copyright Complaints Stanford University Stanford California 94305